packages/twenty-front/src/modules/app/components/App.tsx

```diff
@@ -24,7 +24,7 @@ export const App = () => {
       <RecoilURLSyncJSON location={{ part: 'queryParams' }}>
         <AppErrorBoundary>
           <I18nProvider i18n={i18n}>
-            <CaptchaProvider>
+            <Captcha>
```

Comment:
Agree it's confusing naming but I feel this should be CaptchaProvider to be consitent with UserProvider, ...